### DISCUSSION TITLE
Help with profilename

### DISCUSSION THREAD

**Discussion by deiv061** [schwabe/ics-openvpn#1503] [Q&A]
Hello,

I'm a newbie with OpenVPN configurations.I was able to put everything on one ovpn file as the documentation suggests. Seems everything works as expected but I manually created and import the profile. Can I crate via ADB shell or programmatically the Profile Name with proper configurations for example after reboot to reconnect or let's use only TCP and so on. I saw it can be loaded the profile from the documentation. But I need some automation to create a profile and loaded it.

Thank you!

**Comment by schwabe** [comment]
Yes you can create a profile programmactically. See the remoteExample in the repo. That is a simple program that does exactly that.
